app/src/main/java/ch/epfl/sweng/eventmanager/ui/event/interaction/fragments/EventFeedbackFragment.java

```diff
@@ -88,4 +96,55 @@ public void onAttach(Context context) {
         UNIQUE_ID_DEVICE = Settings.Secure.getString(context.getContentResolver(), Settings.Secure.ANDROID_ID);
         super.onAttach(context);
     }
+
+    public class RatingsAdapter extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
```

Comment:
`RatingsWHATAdapter` ? Looks like an adapter for a RecyclerView-based list... `RatingsAdapter` is too generic.